ddtrace/llmobs/telemetry/constants.py

```diff
@@ -0,0 +1,16 @@
+from enum import Enum
+
+
+LLMOBS_TELEMETRY_METRIC_NAMESPACE = "llmobs"
+
+
+class LLMOBS_TELEMETRY(str, Enum):
+    ACTIVATE_DISTRIBUTED_HEADERS = "activate_distributed_headers"
+    ANNOTATIONS = "annotations"
+    EXPORT_SPAN = "export_span"
+    INIT_TIME = "init_time"
+    INJECT_DISTRIBUTED_HEADERS = "inject_distributed_headers"
+    SPANS_CREATED = "spans_created"
+    SPANS_FINISHED = "spans_finished"
```

Comment:
Should it be singular `span_created/finished`?

Reply:
plural based on the metrics i [defined](https://github.com/DataDog/dd-go/pull/149864/files#diff-974a6d4321da6770ffce5c3a8618b4a6689edd689484debefd3bfd6c57d30366) in the telemetry intake